install/index.markdown

```diff
@@ -15,6 +15,8 @@ MoveIt! is currently released into ROS **Indigo** and **Jade**. ROS **Kinetic**
 
 **Developers:** see [source install](source/) instructions.
 
+**Virtual Machines:** We do not recommend usage of any Rviz-based software, like MoveIt!, with VirtualBox, VMware, etc due to many reports of bugs in virtual graphics.
```

Comment:
I do not consider MoveIt to be "Rviz-based".

How about something along the line of
> There have been many reports of problems with the ROS visualization tool RViz when used in virtual machines.
> We therefore discourage the use of virtual machines unless you are not interested in visualization support.

?